regular-expressions/introduction/basic-syntax-i/regex-wildcards-ii.md

```diff
@@ -53,7 +53,7 @@ We could use the following regex pattern to match these three strings:
 
 We could match just the numbers in the following strings by using this regex:
 
-`/\d+/`
+`/\d/`
 
 ```
 'abc**123**xyz' ✅
```

Comment:
so in this case, `/\d/` will match `1`, `2`, `3` individually, but it won't match `123`. 

we should say what we are matching (either `1`, `2`, `3` individually or `123` as a group)

Reply:
Will add in some text to make this clearer